lte/gateway/docker/services/c/Dockerfile

```diff
@@ -169,34 +169,34 @@ ENV TZ=Europe/Paris
 
 # Install runtime dependencies
 RUN apt-get update && apt-get install -y \
-  apt-utils \
   apt-transport-https \
+  apt-utils \
   ca-certificates \
-  wget \
   gnupg \
-  sudo \
-  netcat \
-  libyaml-cpp-dev \
+  iproute2 \
+  iptables \
   libgoogle-glog-dev \
-  libprotoc-dev \
   libmnl-dev \
+  libprotoc-dev \
   libsctp-dev \
-  psmisc \
-  openssl \
+  libtspi1 \
+  libyaml-cpp-dev \
   net-tools \
+  netcat \
+  openssl \
+  psmisc \
+  sudo \
   tshark \
   tzdata \
-  iproute2 \
-  iptables \
-  libtspi1 \
+  wget \
   && rm -rf /var/lib/apt/lists/*
 
 RUN echo "deb https://artifactory.magmacore.org/artifactory/debian-test focal-ci main" > /etc/apt/sources.list.d/magma.list
 RUN wget -qO - https://artifactory.magmacore.org:443/artifactory/api/gpg/key/public | apt-key add -
 RUN apt-get update && apt-get install -y \
   libopenvswitch \
-  openvswitch-datapath-dkms \
   openvswitch-common \
+  openvswitch-datapath-dkms \
   openvswitch-switch
```

Comment:
Do we want a `&& rm -rf /var/lib/apt/lists/*` here?

Reply:
I added it.